book/B-embedding-git/sections/go-git.asc

```diff
@@ -0,0 +1,84 @@
+=== go-git
+
+(((go-git)))((("Go")))
+In case you want to integrate Git into a service written in Golang, there also is a pure Go library implmentation. In contrast to using a Libgit2 bindings, this implementation does not have any native dependencies, not prone to memory management errors and is transparent for standard Golang performance analysis tooling like CPU, Memory profilers, race detector, etc.
+
+go-git is focused on extensibility, compatibility and supports most of the plumbing APIs, wich is documented https://github.com/src-d/go-git/blob/master/COMPATIBILITY.md[]
+
+Here is a basic example of using Go APIs
```

Comment:
Usually when we have a sentence introducing an example, we end it with a period or a colon. This seems like a great place for a colon.